er_aws_rds/input.py

```
@@ -276,28 +276,20 @@ def is_read_replica(self) -> bool:
         return self.replica_source is not None or self.replicate_source_db is not None
 
     @model_validator(mode="after")
-    def enhanced_monitoring_attributes_require_enhanced_monitoring(self) -> "Rds":
-        """If monitoring_interval is set, enhanced_monitoring must be enabled"""
-        if not self.enhanced_monitoring and (
-            self.monitoring_interval != 0 or self.monitoring_role_arn
-        ):
-            raise ValueError(
-                "Enhanced monitoring attributes requires enhanced_monitoring to be true"
-            )
-        return self
+    def enhanced_monitoring_attributes(self) -> "Rds":
+        """
+        Enhanced monitoring validation:
 
-    @model_validator(mode="after")
-    def monitoring_role_arn_requires_monitoring_interval(self) -> "Rds":
-        """If monitoring_role_arn is set, monitoring_interval must be != 0"""
-        if self.monitoring_role_arn and self.monitoring_interval == 0:
-            raise ValueError("monitoring_role_arn requires a monitoring_interval != 0")
-        return self
+        * If em is disabled, related parameters are removed.
+        * If em is enabled and no monitoring_inverval specificied, set the default value (60)
+        """
+        if not self.enhanced_monitoring:
+            self.monitoring_interval = None
+            self.monitoring_role_arn = None
+
+        if self.enhanced_monitoring and not self.monitoring_interval:
+            self.monitoring_interval = 60
```

this will override `monitoring_interval` to `60` if it was set to `0` explicitly, which conflict with `enhanced_monitoring`, should we raise an exception for it? Or should we deprecate `enhanced_monitoring`, use `monitoring_interval` as single source of truth.

---

Let's stick to `enhanced_monitoring`. Some rds manifests rely on tf-resources to set the default `monitoring_iterval`, so we must do the same. The override will happen only if enhanced_monitoring is true, which makes sense. 

There is only 1 case of monitoring_interval=0 and I'll fix it.

---

PTAL.